source/includes/promo-atlas-fts.rst

```diff
@@ -0,0 +1,14 @@
+.. admonition:: Atlas Full Text Search
+   :class: note
+
+   `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?jmp=docs>`_
+   Full-Text Search and Full-Text Indexes leverage Apache Lucene to
```

Comment:
Maybe just `Full-Text Search Indexes` instead of `Full-Text Search and Full-Text Indexes`